src/ansys/units/systems.py

```diff
@@ -115,8 +126,8 @@ def _set_type(
         setattr(self, f"_{unit_type.name}", unit)
 
     @property
-    def base_units(self):
-        """Units associated with the unit system."""
+    def base_units(self) -> list[str]:
```

Comment:
Thanks for documenting the return type.

To be consistent with other interfaces we ought to return a `dict` here, where we would very clearly map between types and units.

Reply:
TODO figure out if we need this, and how it's used, and whether it should be changed or not.